projects.go

```diff
@@ -106,6 +106,7 @@ type Project struct {
 	PagesAccessLevel                          AccessControlValue         `json:"pages_access_level"`
 	OperationsAccessLevel                     AccessControlValue         `json:"operations_access_level"`
 	AutocloseReferencedIssues                 bool                       `json:"autoclose_referenced_issues"`
+	SuggestionCommitMessage                   string                     `url:"suggestion_commit_message,omitempty" json:"suggestion_commit_message,omitempty"`
```

Comment:
The url tag and `omitempty` is not needed in this struct:

```suggestion
	SuggestionCommitMessage                   string                     `json:"suggestion_commit_message"`
```